### PR TITLE
metrics: Update cassandra to support block volumes

### LINKED ISSUE #4842: metrics: update cassandra test to support block volumes

### PR DESCRIPTION
**Pull request by GabyCT**
This PR updates the cassandra kuberentes test to support block volumes.

Fixes #4842

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: metrics/disk/cassandra_kubernetes/cassandra.sh [hunk 1]
```diff
@@ -125,21 +125,51 @@ function cassandra_start() {
 	wait_time=20
  	sleep_time=2
 
+	vol_capacity="500M"
```
Comment (amshinde) [inline metrics/disk/cassandra_kubernetes/cassandra.sh:128]: Is this capacity enough for the cassandra workload? If I recall the stateful set was using at least 1G with three replicas. 
I would bump this to 3G atleast,

File: metrics/disk/cassandra_kubernetes/cassandra.sh [hunk 2]
```diff
@@ -125,21 +125,51 @@ function cassandra_start() {
...
 	kubectl create -f "${SCRIPT_PATH}/runtimeclass_workloads/cassandra-service.yaml"
 
 	# Check service
 	kubectl get svc | grep "$service_name"
 
-	# Create local volumes
-	kubectl create -f "${SCRIPT_PATH}/runtimeclass_workloads/local-volumes.yaml"
-
-	# Create statefulset
-	kubectl create -f "${SCRIPT_PATH}/runtimeclass_workloads/cassandra-statefulset.yaml"
```
Comment (amshinde) [inline metrics/disk/cassandra_kubernetes/cassandra.sh:138]: Why is the stateful set not being used anymore?

File: metrics/disk/cassandra_kubernetes/volume/local-storage.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: local-storage
```
Comment (amshinde) [inline metrics/disk/cassandra_kubernetes/volume/local-storage.yaml:4]: Can we change the name here to block-local-storage to be clear and similarly update at all other places. 
My idea is we could augment the test in the near future both to test block as well as virtio-fs.
For testing virtio-fs we could use another storage class setup using host volumes. What do you think?

File: metrics/disk/cassandra_kubernetes/cassandra.sh (hunk 2 above)
Comment (GabyCT) [reply metrics/disk/cassandra_kubernetes/cassandra.sh:138]: @amshinde yes it is being used see https://github.com/kata-containers/tests/blob/9822556eb79afac3f8f1ca4bbe169850dc3b10f6/metrics/disk/cassandra_kubernetes/cassandra.sh#L165

File: metrics/disk/cassandra_kubernetes/cassandra.sh (hunk 1 above)
Comment (GabyCT) [reply metrics/disk/cassandra_kubernetes/cassandra.sh]: @amshinde changes applied

File: metrics/disk/cassandra_kubernetes/volume/local-storage.yaml (hunk 3 above)
Comment (GabyCT) [reply metrics/disk/cassandra_kubernetes/volume/local-storage.yaml]: @amshinde changes applied

**Review by amshinde — APPROVED** [review]

**Review by dborquez — APPROVED** [review]
LGTM  thanks @GabyCT

**Pull request merged**